### DISCUSSION TITLE
kneeling on my knees

### DISCUSSION THREAD

**Discussion by ident444** [mozilla/hubs#5314] [General Questions & Discussion]
Hi there, I'm an absolute beginner in hubs, maybe this is a stupid question: when I'm walking through the room, my view is, as if I would kneeling on my knees. I would like to increase my height to have a better overview. What should I do? Thanks for helping.

**Comment by pattersonbl2** [comment]
Could you share a screen shoot of this?

**Comment by matthewbcool** [comment]
Hi @ident444, are you using a custom avatar? note the avatars height could potentially be the issue you are facing. https://hubs.mozilla.com/docs/creators-advanced-avatar-customization.html#create-your-own-avatar

If you share the avatar you are using and the scene you are experiencing this issue in, we can dig more into the cause.
